yt/visualization/tests/test_color_maps.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+import matplotlib
+import numpy as np
+from nose.tools import assert_raises
+
+from yt import show_colormaps, make_colormap
+from yt.testing import assert_equal, assert_almost_equal
+
+matplotlib.use('Agg')
+
+class TestColorMaps(unittest.TestCase):
+
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+        self.curdir = os.getcwd()
+        os.chdir(self.tmpdir)
+
+    def tearDown(self):
+        os.chdir(self.curdir)
+        shutil.rmtree(self.tmpdir)
+
+    def test_show_colormaps(self):
+        show_colormaps()
+        show_colormaps(subset=["jet", "cool"])
+        show_colormaps(subset="yt_native", filename="yt_color_maps.png")
+
+        # Test for non-existent color map
+        with assert_raises(Exception) as ex:
```

Comment:
Does this really raise a plain `Exception` class? If so perhaps we should be using a more explicit `Exception` subclass. If not, please test for the exact exception class here.